src/containers/AccountConnection.jsx

```diff
@@ -13,13 +13,14 @@ class AccountConnection extends Component {
   constructor (props, context) {
     super(props, context)
     this.store = this.context.store
+    this.state = {
+      account: this.props.existingAccount
```

Comment:
In my opinion, existingAccount should not change.
First, I thank about passing a new connected account to the parent component and let him update `AccountConnection`'s props. But the result was that as soon that a new account was created, the form was mutating to the edition form, still submitting data. It's pretty much an UI issue, but we need here to keep the fact that an account is a new created one or an existing one.
But I am totally open to a better way to handle this, actually the current way does not satisfy me totally.